### PR TITLE
Added authorization.

### PR DESCRIPTION
**Pull request by SBanczyk**

### CODE REVIEW

File: compose.yaml [hunk 1]
```diff
@@ -1,8 +1,29 @@
...
+      - SECRET_KEY=${SECRET_KEY}
+      - AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID}
+      - AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY}
+      - REGION_NAME=${REGION_NAME}
+      - DB_IP=${DB_IP}
+      - DB_NAME=${DB_NAME}
+      - DB_USER=${DB_USER}
+      - DB_PWD=${DB_PWD}
+volumes:
+  pgdata:
```
Comment (domel8dp) [inline compose.yaml]: maybe this is the reason that pgdata directory remains empty

File: aws_connect.py [hunk 2]
```diff
@@ -1,28 +1,173 @@
...
+    else:
+        break
+with db_connect.cursor() as cursor:
+    cursor.execute("SELECT * from users")
+    row = cursor.fetchone()
+    while row is not None:
+        user_db = {
+            row[0]: {"username": row[0], "hashed_password": row[1], "disabled": row[2]}
+        }
+        row = cursor.fetchone()
```
Comment (domel8dp) [inline aws_connect.py]: we do not need to keep the user database in program memory, we can look up user only when needed (in `get_user`)

File: aws_connect.py [hunk 3]
```diff
@@ -1,28 +1,173 @@
...
+    current_user: Annotated[User, Depends(get_current_active_user)],
+):
+    return current_user
+
+
+@app.get("/users/me/items/")
+async def read_own_items(
+    current_user: Annotated[User, Depends(get_current_active_user)],
+):
+    return [{"item_id": "Foo", "owner": current_user.username}]
```
Comment (domel8dp) [inline aws_connect.py]: this is an example endpoint, not needed in this application

**Review by domel8dp — CHANGES_REQUESTED** [review]

File: compose.yaml (hunk 1 above)
Comment (SBanczyk) [reply compose.yaml]: even after deleting that, the directory is still empty

**Review by domel8dp — APPROVED** [review]

**Pull request merged**